File: homework-1.java

public class Homework1 {
    public static void main(String[] args) {
      String item = args[0];
      String price = args[1];
      Integer.parseInt(price);
      String time = args[2];
      Integer.parseInt(time);
      int interest = 5 + (int)(Math.random() * ((15) + 1));
      Integer.parseInt(interest);
      int total = ((interest*time + 100.0)/100.0) * price;
      int remainder = total - price;
      System.out.println("The " + item + " with a loan of " + price + " after "
      + time + " with a rate of " + interest + " will cost " + total +
      "with an interest of " + remainder);

    }
}

File: homework-1.rb
item = ARGV[0]
price = ARGV[1].to_f/1.0
#5-20
interest = rand(5..20).to_i
time = ARGV[2].to_i
total = ((((100.0+interest)/100.0)**time)*price*100).round / 100.0
remainder = ((total - price)*100).round / 100.0
puts "The total amount you would be paying for the #{item} after #{time}
years with a loan of $#{price} and an interest rate of #{interest}% would be
$#{total} and the interest is $#{remainder}."
